dev-libs/wlc/metadata.xml

```diff
@@ -9,8 +9,4 @@
 		<email>proxy-maint@gentoo.org</email>
 		<name>Proxy Maintainers</name>
 	</maintainer>
-	<use>
-		<flag name="systemd">Enable support for systemd-logind.</flag>
-		<flag name="X">Enable X11 backend and XWayland support.</flag>
```

Comment:
Just to be clear, it's fine to include multiple changes in a single pull request. However, since we're using git, it would be a good idea to split it into a separate commit. This way it's easier to follow changes, rationale and revert if necessary.